tox.ini

```diff
@@ -28,11 +28,6 @@ commands=
     dot -Tpng -odocs/images/action.png action.dot
     sphinx-build -b html -d docs/_build docs docs/_build/html
 
-[testenv:itest_deb]
```

Comment:
Remove from travis, then?

Reply:
already done, travis is now using make target with the same name